HomeTask3/Program.cs

```diff
@@ -0,0 +1,129 @@
+﻿var task = new MyHomework();
+//task.RunTask2();
+
+while (task.RunTask1())
+{
+
+}
+
+while (task.RunTask2())
+{
+
+}
+
+while (task.RunTask3())
+{
+
+}
+
+public class MyHomework
+{
+
+    public bool RunTask1()
+    {
+        Console.WriteLine("Enter the number:");
+        var numberFromConsole = Console.ReadLine();
+
+        if (numberFromConsole.ToLower() == "exit")
+        {
+            return false;
+        }
+
+
+        CheckNumber(int.Parse(numberFromConsole));
+
+        return true;
+    }
+
+    public bool RunTask2()
+    {
+        Console.WriteLine("Enter 2 numbers:");
+        var FirstnumberFromConsole = Console.ReadLine();
+        var SecondnumberFromConsole = Console.ReadLine();
+        if (FirstnumberFromConsole.ToLower() == "exit" || SecondnumberFromConsole.ToLower() == "exit")
+        {
+            return false;
+        }
+
+        int Firstnumber = int.Parse(FirstnumberFromConsole);
+        int Secondnumber = int.Parse(SecondnumberFromConsole);
+
+        for (var i = Firstnumber; i <= Secondnumber; i++)
+        {
+            CheckNumber(i);
+        }
+
+        return true;
+
+    }
+
+    public bool RunTask3()
+    {
+        Console.WriteLine("Enter 2 numbers:");
+        var FirstnumberFromConsole = Console.ReadLine();
+        var SecondnumberFromConsole = Console.ReadLine();
+        if (FirstnumberFromConsole.ToLower() == "exit" || SecondnumberFromConsole.ToLower() == "exit")
+        {
+            return false;
+        }
+        int Firstnumber = int.Parse(FirstnumberFromConsole);
+        int Secondnumber = int.Parse(SecondnumberFromConsole);
+
+        if (Firstnumber < Secondnumber)
+        {
+            for (var i = Firstnumber; i <= Secondnumber; i++)
+            {
+                CheckNumber(i);
+            }
+
+        }
+        else
+        {
+            for (var i = Secondnumber; i <= Firstnumber; i++)
+            {
+                CheckNumber(i);
+            }
+        }
+
+        return true;
+    }
+
+
+    protected void CheckNumber(int number)
+    {
+        bool isDividedByTwo = IsDividedByTwo(number);
+        bool isDividedByFive = IsDividedByFive(number);
+
+        if (isDividedByTwo && isDividedByFive)
+        {
+            Console.WriteLine("tuttu-frutti");
+        }
+        else
```

Comment:
no need to use nested constructions. It would be enough to use if...else if...else if

Reply:
ok